src/AuthenticationService.php

```diff
@@ -227,11 +227,14 @@ public function clearIdentity(ServerRequestInterface $request, ResponseInterface
     {
```

Comment:
Here it still seems to be `@return \Psr\Http\Message\ServerRequestInterface` instead of array.